### PR TITLE
Linting update

### PR DESCRIPTION
**Pull request by fclairamb**

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -191,6 +193,20 @@ func (server *FtpServer) Listen() error {
 	return err
 }
 
+func temporaryError(err net.Error) bool {
```
Comment (fclairamb) [inline server.go:196]: @drakkan this is the biggest change. What do you think about it?

Comment (drakkan) [reply server.go:196]: @fclairamb in SFTPGo I have the [same code](https://github.com/drakkan/sftpgo/blob/main/internal/sftpd/server.go#L355) for the SFTP server. I have added a lint exclusion for now. I will update the SFTPGo code once the Go developers update that of the http server. 
I think it is also ok if you want to handle accept errors directly in ftpserverlib, they are very special cases, I don't think I've ever seen that type of error I just copied the http server code

File: server.go [hunk 2]
```diff
@@ -191,6 +193,20 @@ func (server *FtpServer) Listen() error {
...
+	if opErr, ok := err.(*net.OpError); ok {
+		if sysErr, ok := opErr.Err.(*os.SyscallError); ok {
+			if errno, ok := sysErr.Err.(syscall.Errno); ok {
+				if errno == syscall.ECONNABORTED || errno == syscall.ECONNRESET {
+					return true
```
Comment (drakkan) [inline server.go:202]: @fclairamb please note this

https://github.com/golang/go/blob/master/src/net/error_unix.go
https://github.com/golang/go/blob/master/src/net/error_windows.go

`syscall.ECONNABORTED` and `syscall.ECONNRESET` are probably undefined on Windows

File: server.go (hunk 1 above)
Comment (fclairamb) [reply server.go:196]: I'll keep this `temporaryError` method but revert to the previous code to avoid creating regressions.

**Pull request merged**